cruise/CruiseInformation.csv

```diff
@@ -15,46 +17,60 @@ KN-222,R/V Knorr,2014-10-03T22:00:00,2014-10-11T11:00:00,Pioneer 3 Leg 1
 OC1410A,R/V Oceanus,2014-10-06T00:00:00,2014-10-10T23:59:59,Endurance 2 (rvdata)
 KN-224,R/V Knorr,2014-12-11T19:00:00,2014-12-16T23:59:59,Pioneer 3 leg 2
 WJ20141230,M/V Warren Jr.,2014-12-30T13:30:00,2014-12-31T12:00:00,Pioneer ISSM-1 buoy recovery
-SI-1501,M/V Scarlett Isabella,2015-01-10T00:00:00,2015-01-13T23:59:59,Pioneer 3a
+SI-1501,M/V Scarlett Isabella,2015-01-10T00:00:00,2015-01-13T23:59:59,#TO BE DELETED
+SI20150110,M/V Scarlett Isabella,2015-01-10T00:00:00,2015-01-13T23:59:59,Pioneer 3a
 PS1501,R/V Pacific Storm,2015-01-21T20:28:00,2015-01-21T22:02:00,Endurance GL info
 AT-26-29,R/V Atlantis,2015-02-12T15:00:00,2015-03-04T23:59:59,Southern Ocean 1
+EN552,R/V Endeavor,2015-03-01T00:00:00,2015-03-05T23:59:59,Pioneer glider cp_387 recovery
+MM20150327,R/V Maria S. Merian,2015-03-27T00:00:00,2015-03-27T23:59:59,Pioneer glider cp_336 recovery
 AT-26-30,R/V Atlantis,2015-03-08T12:00:00,2015-03-26T23:59:59,Argentine Basin 1
 EK-1503,R/V Elakha,2015-03-18T00:00:00,2015-03-18T23:59:59,Endurance CE02SHSP
 OC1503D,R/V Oceanus,2015-04-02T00:00:00,2015-04-12T23:59:59,Endurance 3a (rvdata)
 AT-27,R/V Atlantis,2015-04-28T16:00:00,2015-05-14T12:26:00,Pioneer 4
 TN-323,R/V Thomas G. Thompson,2015-05-29T16:00:00,2015-06-13T04:00:00,Papa 3
 PS1506A,R/V Pacific Storm & R/V Elakha ,2015-06-03T00:00:00,2015-06-03T23:59:59,Endurance 3b
 EK-1506,R/V Elakha,2015-06-23T00:00:00,2015-06-23T23:59:59,Endurance CE01ISSP
+MH20150702,R/V Matthew Hughes,2015-07-02T00:00:00,2015-07-02T23:59:59,Pioneer glider cp_340 recovery

```

Comment:
Not that it matters, but this was 340 and 379 recovery

Reply:
There will be a few cases like that where only one glider is listed in the comment but multiple were recovered on the cruise.  Whichever glider was reviewed first and led me to adding the cruise to this csv got named in the comment. If we think it's important we could add multiple glider names to the comments

Reply:
It seems inconsistent across arrays and cruises. For recoveries (of gliders, or platforms adrift), there is not a lot of documents (no cruise plan, there may be no quick look, etc).